Gateway/Http/Client/TransactionPosCloudSync.php

```diff
@@ -108,7 +111,7 @@ public function placeRequest(TransferInterface $transferObject): array
         $terminalId = $request['terminalID'];
 
         if (array_key_exists('chainCalls', $request)) {
-            $quote = $this->initiatePosPayment($terminalId, $request['numberOfInstallments']);
+            $quote = $this->initiatePosPayment($terminalId, $request['fundingSource'], $request['numberOfInstallments']);
```

Comment:
Should we check if the `fundingSource` exists ?

Reply:
Hey @AlexandrosMor, thanks for the review! We don't need to check, because it is set by the knockout observer to `credit` by default.

Reply:
Cool, thanks!